package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Type definitions for Corvid by Wix",
   "scripts": {
     "lint": "eslint ./src/** ./test/** ./scripts/**",
-    "build": "npm run generate-dts && npm run patch-declarations && npm run generate-full-corvid-types && npm run generate-module-list && npm run generate-event-handlers && npm run types && webpack",
+    "build": "npm run generate-dts && npm run patch-declarations && npm run generate-full-corvid-types && npm run generate-module-list && npm run generate-event-handlers && webpack && npm run types",
     "test": "npm run lint && jest",
-    "types": "tsc src/types.ts --declaration --emitDeclarationOnly --outDir dist",
+    "types": "tsc -p ./tsconfig.json --emitDeclarationOnly",
```

Comment:
why having flags here and not in the `tsconfig` as well?

Reply:
also is `-p tsconfig.json` really needed? isn't that the default convention?

Reply:
1. Having the flag here means that we can lint, since it is emitting. But when we build we only want the declarations.
2. This is because of how this library is used. docworks clones corvid-types locally from github and run tsc. If docworks is ran from a project that already has a tsconfig, things get weird as it looks for configs upwards and includes unrelated files.